test/csv/csv-filter/test_csv_filter.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: false
+
+require_relative '../helper'
+
+require 'csv'
+
+class TestFilter < Test::Unit::TestCase
+
+  # Some rows data (useful as default).
+  Rows = [
+    %w[aaa bbb ccc],
+    %w[ddd eee fff],
+  ]
+
+  # Return CSV string generated from rows array and options.
+  def make_csv_s(rows: Rows, **options)
+    CSV.generate(**options) do|csv|
+      rows.each do |row|
+        csv << row
+      end
+    end
+  end
+
+  # Return filepath of file containing CSV data.
+  def csv_filepath(csv_in_s, dirpath, option_sym)
+    filename = "#{option_sym}.csv"
+    filepath = File.join(dirpath, filename)
+    File.write(filepath, csv_in_s)
+    filepath
+  end
+
+  # Return stdout and stderr from CLI execution.
+  def run_csv_filter(filepath, cli_option_names = [])
```

Comment:
How about using variable length arguments for easy to use?

```suggestion
  def run_csv_filter(filepath, *cli_option_names)
```